src/components/Score.js

```diff
@@ -4,16 +4,41 @@ import { browserHistory } from 'react-router';
 import ScoreInput from './ScoreInput.js';
 
 const Score = React.createClass({
-  submitScores() {
-    browserHistory.push('/d/' + this.props.params.sessionId);
+  getInitialState() {
+    return {
+      scores: {}
+    };
+  },
+  saveScore() {
+    // TODO: validation - pk2
+    const data = JSON.stringify({
+      sessionId: this.state.sessionId,
+      scopes: this.state.scores
+    });
+    fetch('http://localhost:3000/api/score', {
```

Comment:
Thanks for nudging on this, looks like we had it installed, but were never using it. 

Update coming to this effect: https://gist.github.com/Couto/b29676dd1ab8714a818f#gistcomment-1584602